R/auto.R

```diff
@@ -31,6 +31,7 @@ pre_release_impl <- function(which, force) {
 
   cat(boxx("pre-release", border_style = "double"))
 
+  # not sure how to do this in gert
   stopifnot(git2r::is_branch(git2r::repository_head()))
```

Comment:
Parhaps:

```suggestion
  # https://github.com/r-lib/gert/issues/139
  stopifnot(gert::git_branch() != "HEAD")
```

Reference: https://github.com/r-lib/gert/issues/139